hooks/common/post-db-copy/update.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+path=`dirname "$0"`
```

Comment:
And instead of using `dirname "$0"`, we can consistently use
$(dirname "$0") and wrap it with double quotes.  It is preferred to to use $() than ` `.